openstudiocore/src/utilities/data/TimeSeries.cpp

```diff
@@ -626,11 +765,40 @@ namespace openstudio{
 
     std::shared_ptr<TimeSeries_Impl> TimeSeries_Impl::operator*(double d) const {
 
-      return std::shared_ptr<TimeSeries_Impl>(new TimeSeries_Impl(m_firstReportDateTime, 
-        m_secondsFromFirstReport,
-        m_values*d, 
+      return std::shared_ptr<TimeSeries_Impl>(new TimeSeries_Impl(m_startDateTime,
+        m_secondsFromStart,

```

Comment:
@macumber You are correct. This is left over from when I changed the interface. If we do get rid of some of the internals (m_secondsFromFirstReport, for example) then we will need to think about changing the impl interface some. We can do that, right?